test/snippet/alphabet/aminoacid/aa20_construction.cpp

```diff
@@ -0,0 +1,17 @@
+#include <seqan3/alphabet/aminoacid/aa20.hpp>
+#include <seqan3/core/debug_stream.hpp>
+
+int main()
+{
+    using seqan3::operator""_aa20;
+    seqan3::aa20 my_letter{'A'_aa20};
+    // doesn't work:
+    // aa20 my_letter{'A'};
```

Comment:
as already mentioned by someone else, this is a good opportunity to get rid of all the `doesn't work` snippets.

Reply:
(It only exists because we didn't have general alphabet documentation in the beginning and then everybody copy'n'pasted it from my first alphabet)